### ISSUE TITLE
Dependency is running very old version of lodash

### ISSUE THREAD

**Issue by paulsjv** [tniswong/web-build#13]
One of your dependencies is using a very old version of lodash.

`npm WARN deprecated lodash@1.0.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0`

**Comment by tniswong** [comment]
Yeah, I've noticed that. I've not spent the time to investigate the culprit, as it doesn't seem to be causing any problems.
